_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,117 @@
+published with us and JOSS. This, friends, is a win-win!
+
+## Who do we serve?
+pyOpenSci supports maintainers of smaller scientific python open source projects. 
+We also support scientists using these tools and will empower folks like you 
+with the skills needed to both contribute to Open Source and improve your open 
+science workflows! In the long term these skills will be invaluable in your 
+data intensive career trajectory!
+
+## pyOpenSci is about collaboration - not reinventing the wheel
+Collaborations with existing efforts are critical. I welcome any feedback from 
+you regarding groups that we should be engaging with! I will also need help 
+finding reviewers and editors, mentors to support training efforts and voices to 
+spread the word about the project.
+
+## And… what's the timeline?
+Unless I figure out how to create numerous robot clones of myself, please know 
+that all of the above won’t happen immediately. I am currently the sole 
+employee of the organization and this is my second week. But never fear - we 
+will accomplish a lot in the next two years and look forward to and will greatly
+value support from some of you along the way!
+
+## Stay in touch!
+There are many ways to stay in touch with our project. Once I have a road map 
+for activities developed, I will communicate this here on our 
+[website](https://www.pyopensci.org/), Twitter 
+and discourse. So be sure to follow [@pyOpenSci on Twitter](https://www.twitter.com/pyopensci). 
+
+## What can you do now?
+There are a few ways to get involved with us now.
+
+1. [You can sign up to be a reviewer by filling out our reviewer form](https://forms.gle/B6zAukLCvJot5nws6). If you do this, we will reach out to you if a package is submitted that suits your background and interests!
+  * We will also be growing our editorial board in the upcoming months. Once you have reviewed for us you are eligible to be considered as an editor. We will be posting more about this process in the upcoming months.
+1. Our open peer review process is running! If you have a package that you are working on, [you can submit it to our peer review process](https://github.com/pyOpenSci/software-review/issues). 
+  * You can learn more about our review process here in our [community guide (which will be updated in the upcoming months so that it is more clear)](https://www.pyopensci.org/contributing-guide/open-source-software-peer-review/aims-and-scope.html).
+  * If you have a package that you wish to submit but need some guidance, you can also submit a presubmission inquiry or a request for help here.
+2. You can [follow us on twitter](https://www.twitter.com/pyopensci) to keep tabs on our progress. 
+
+If you want to stay in touch with me, feel free to follow me on
+([twitter: @leahawasser](https://www.twitter.com/leahawasser)). Or connect with 
+me on [GitHub](https://www.github.com/lwasser). 
+
+## pyOpenSci was built with support from numerous community members
```

Comment:
I went thru meeting notes to grab names. i KNOW i'm missing some names as not everyone signed in and i'm sure i'm just missing people so let meknow if you think this is dangerous. i just want to acknowledge people.